### PR TITLE
merge testnet, update contract-sdk

### PR DESCRIPTION
**Pull request by lamcc21**
- update contract sdk
- add docker-compose, project.yaml
- update script
- update contract-sdk

### CODE REVIEW

File: project-kepler.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version: 1.0.0
 runner:
   node:
     name: '@subql/node-ethereum'
-    version: '*'
+    version: '>2.8.0'
```
Comment (ianhe8x) [inline project-kepler.yaml:7]: doesn't seems this project rely on api.getNetwork

Comment (ianhe8x) [inline project-kepler.yaml:7]:
```suggestion
    version: '*'
```

File: project-testnet.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+specVersion: 1.0.0
+name: subquery-airdrop-project
+version: 1.0.0
+runner:
+  node:
+    name: '@subql/node-ethereum'
+    version: '>2.8.0'
```
Comment (ianhe8x) [inline project-testnet.yaml:7]:
```suggestion
    version: '*'
```

File: project-testnet.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+repository: 'https://github.com/subquery/airdrop-subql-project.git'
+schema:
+  file: ./schema.graphql
+network:
+  chainId: '80001' # Mumbai testnet 
+  endpoint: 'https://polygon-mumbai.g.alchemy.com/v2/olBcbaUN-00iYo-4VDlJYbkpCmj5rLY0'
+
+x-ethereum: &ethereum
+  kind: ethereum/Runtime 
+  startBlock: 33168664
```
Comment (ianhe8x) [inline project-testnet.yaml:23]: need to update this to the new height

**Review by ianhe8x — CHANGES_REQUESTED** [review]

File: project-testnet.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  startBlock: 37528625
+  assets:
+    airdropper:
+      file: ./node_modules/@subql/contract-sdk/artifacts/contracts/Airdropper.sol/Airdropper.json
+
+dataSources:
+  - <<: *ethereum
+    options:
+      abi: airdropper
+      address: '0xB64D73B96358855075576D66746D2a88e043CC1E'
```
Comment (mzxyz) [inline project-testnet.yaml:32]: Update the airdrop contract address: `0xdE4c8a50A8AFC4BcB9250Df8f4d61C8eBF98F535`

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**